sdk/synapse/synapse-artifacts/review/synapse-artifacts.api.md

```diff
@@ -4487,74 +4725,68 @@ export interface LinkedService {
-}
-
-// @public
-export interface LinkedServiceOperationsCreateOrUpdateLinkedServiceOptionalParams extends coreClient.OperationOptions {
+export interface LinkedServiceCreateOrUpdateLinkedServiceOptionalParams extends coreClient.OperationOptions {
     ifMatch?: string;
     resumeFrom?: string;
     updateIntervalInMs?: number;
 }
 
 // @public
-export type LinkedServiceOperationsCreateOrUpdateLinkedServiceResponse = LinkedServiceResource;
+export type LinkedServiceCreateOrUpdateLinkedServiceResponse = LinkedServiceResource;
 
 // @public
-export interface LinkedServiceOperationsDeleteLinkedServiceOptionalParams extends coreClient.OperationOptions {
+export type LinkedServiceDebugResource = SubResourceDebugResource & {
+    properties: LinkedServiceUnion;
+};
+
+// @public
+export interface LinkedServiceDeleteLinkedServiceOptionalParams extends coreClient.OperationOptions {
     resumeFrom?: string;
     updateIntervalInMs?: number;
 }
 
 // @public
-export interface LinkedServiceOperationsGetLinkedServiceOptionalParams extends coreClient.OperationOptions {
+export interface LinkedServiceGetLinkedServiceOptionalParams extends coreClient.OperationOptions {
     ifNoneMatch?: string;
 }
 
 // @public
-export type LinkedServiceOperationsGetLinkedServiceResponse = LinkedServiceResource;
+export type LinkedServiceGetLinkedServiceResponse = LinkedServiceResource;
 
 // @public
-export interface LinkedServiceOperationsGetLinkedServicesByWorkspaceNextOptionalParams extends coreClient.OperationOptions {
+export interface LinkedServiceGetLinkedServicesByWorkspaceNextOptionalParams extends coreClient.OperationOptions {
 }
 
 // @public
-export type LinkedServiceOperationsGetLinkedServicesByWorkspaceNextResponse = LinkedServiceListResponse;
+export type LinkedServiceGetLinkedServicesByWorkspaceNextResponse = LinkedServiceListResponse;
 
 // @public
-export interface LinkedServiceOperationsGetLinkedServicesByWorkspaceOptionalParams extends coreClient.OperationOptions {
+export interface LinkedServiceGetLinkedServicesByWorkspaceOptionalParams extends coreClient.OperationOptions {
 }
 
 // @public
-export type LinkedServiceOperationsGetLinkedServicesByWorkspaceResponse = LinkedServiceListResponse;
+export type LinkedServiceGetLinkedServicesByWorkspaceResponse = LinkedServiceListResponse;
 
 // @public
-export interface LinkedServiceOperationsRenameLinkedServiceOptionalParams extends coreClient.OperationOptions {
-    resumeFrom?: string;
-    updateIntervalInMs?: number;
+export interface LinkedServiceListResponse {
+    nextLink?: string;
+    value: LinkedServiceResource[];
+}
+
+// @public
+export interface LinkedServiceOperations {
+    beginCreateOrUpdateLinkedService(linkedServiceName: string, linkedService: LinkedServiceResource, options?: LinkedServiceCreateOrUpdateLinkedServiceOptionalParams): Promise<PollerLike<PollOperationState<LinkedServiceCreateOrUpdateLinkedServiceResponse>, LinkedServiceCreateOrUpdateLinkedServiceResponse>>;
```

Comment:
Ideally, this should be a poller that return an iterator but I take it that you disabled paging for this operation, right? autorest currently can not handle this case unfortunately.